script/create-release.js

```diff
@@ -0,0 +1,24 @@
+const glob = require('glob')
+const path = require('path')
+const fs = require('fs')
+
+module.exports = ({ github, context, artifactsDir, releaseId }) => {
+  glob(artifactsDir + '/**/*', { nodir: true }, async function (err, files) {
+    for (const file of files) {
+      if (file.endsWith('.lzma.sha256')) {
```

Comment:
The `.lzma.sha256` files were excluded in Travis so I excluded them here as well